### PR TITLE
[14.0][ADD]sale_commission_settlement_report: manage settlement report visualization

### PR DESCRIPTION
**Pull request by PicchiSeba**

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

File: sale_commission_settlement_report/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module contains some settings applied to the settlement itself or the report.
```
Comment (francesco-ooops) [inline sale_commission_settlement_report/readme/DESCRIPTION.rst]: This module allows to:

- add invoice customer info to settlement report
- hide lines with amount = 0 from settlement report

File: sale_commission_settlement_report/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+Refer to the Configuration section for the details
```
Comment (francesco-ooops) [inline sale_commission_settlement_report/readme/DESCRIPTION.rst]: Remove

File: sale_commission_settlement_report/readme/CONFIGURE.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+#. Go to *Settings > Sales > Quotations & Orders*:
+
+  * enable the setting "Show partner details in settlements report"
+
+
+To hide lines with amount equals to 0:
+
+#. Go to *Settings > Sales > Quotations & Orders*:
+
+  * enable the setting "Skip lines in settlements with zero amount"
```
Comment (francesco-ooops) [inline sale_commission_settlement_report/readme/CONFIGURE.rst]: * enable the setting "Do not display lines with zero amount in settlements and its report"

**Review by renda-dev — APPROVED** [review]
Code review, LGTM

**Comment by francesco-ooops** [comment]
@pedrobaeza fixed your comment here: https://github.com/OCA/commission/pull/544#pullrequestreview-2191718121

Could you merge? Thanks!

**Pull request merged**